risedev.yml

```diff
@@ -1020,16 +1020,6 @@ profile:
       - use: sqlite
       - use: minio
 
-  ci-iceberg-test:
-    config-path: src/config/ci-iceberg-test.toml
-    steps:
-      - use: minio
-      - use: meta-node
-      - use: compute-node
-        enable-tiered-cache: true
-      - use: frontend
-      - use: compactor
```

Comment:
I don't find any special reason for having a separate risedev profile and a config file 🤔